src/common/logger/ShortlistingLogger.ts

```
@@ -0,0 +1,333 @@
+  private static readonly LOG_DIR = 'logs';
+  /** Name of the CSV file containing failure logs */
+  private static readonly FAILURE_LOG_FILE = 'shortlisting-failures.csv';
+  /** Winston logger instance for shortlisting operations */
+  private static shortlistingLogger: winston.Logger;
+
+  /**
+   * Gets the dedicated shortlisting logger instance
+   * Creates a new logger if it doesn't exist
+   */
+  private static getShortlistingLogger(): winston.Logger {
+    if (!this.shortlistingLogger) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)